integrations/scylladb/prometheus_metadata.yaml

```diff
@@ -70,3 +74,71 @@ platforms:
         kind: CUMULATIVE
         value_type: DOUBLE
     install_documentation_url: https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/scylladb
+  - type: GKE
+    version: '2'
+    launch_stage: GA
+    exporter_metadata:
+      name: ScyllaDB Prometheus Exporter
+      doc_url: https://monitoring.docs.scylladb.com/stable/reference/monitoring_apis.html
+      minimum_supported_version: "5.2"
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/scylla_node_operation_mode/gauge
```

Comment:
Can you update this metric_type to be a metric that is *not* collected by the 5.0 exporter version? This will allow us to differentiate between this version of the exporter and the previous version.